scripts/install/calibre-web.sh

```diff
@@ -0,0 +1,124 @@
+#!/bin/bash
+# Flying Sausages 2020 for swizzin
+#Calibre-web installer
+
+calibrewebdir="/opt/calibre-web"
+calibrewebuser="calibreweb" # or make this master user?
+codename=$(lsb_release -cs)
+
+function _install_dependencies_calibreweb() {
+	apt_install unzip imagemagick
+}
+
+function _install_calibreweb() {
+	echo_progress_start "Creating venv for calibre-web"
+	if [[ $codename =~ ("bionic"|"stretch"|"xenial") ]]; then
```

Comment:
Can you just add a comment regarding the minimum python version required? At some point I'm going to update the logic of the python venvs to use the python version rather than relying on codename

Reply:
Just says "Install dependencies by running pip3 install --target vendor -r requirements.txt (python3.x) or pip install --target vendor -r requirements.txt (python2.7)."

Reply:
```suggestion
    # Minimum version is Python 3.x
	if [[ $codename =~ ("bionic"|"stretch"|"xenial") ]]; then
```

Reply:
If that's the case, is it required to install pyenv on these distros?

Reply:
I honestly wouldn't know. what do you advise? Just install python3 on all releases and make the venv? I'll do that for now, don't see anything wrong with it.

Reply:
I would just use `python 3 -m venv` to create the venv for all distros. Unless the dev states a minimum Python 3 version, installing pyenv just takes extra time when the system packages are good enough. 3.x doesn't really seem to suggest a minimum version

Reply:
I would just use python 3 -m venv to create the venv for all distros. Unless the dev states a minimum Python 3 version, installing pyenv just takes extra time when the system packages are good enough. 3.x doesn't really seem to suggest a minimum version

Reply:
Got rid of the `pyenv`, going straight for just `python3`